stratum/hal/lib/common/utils.cc

```diff
@@ -259,6 +259,19 @@ std::string ConvertAdminStateToString(const AdminState& state) {
   }
 }
 
+bool ConvertLoopbackStateToBool(const LoopbackState& state) {
+  switch (state) {
+    case LOOPBACK_NONE:
+      return false;
+    case LOOPBACK_MAC:
+      return true;
+    case LOOPBACK_PHY:
+      return true;
+    default:
+      return false;
+  }
+}
```

Comment:
do we need both `ConvertLoopbackStateToBool` and `IsLoopbackStateEnabled` ?

Reply:
no, it's a leftover from `ConvertLoopbackStateToBool`